src/doppelkopf.js

```diff
@@ -119,10 +51,15 @@ window.onload = () => {
     beginButton.onclick = initScorepad;
   });
 
+  /**
+   * Redirects the user to the specified scorepad.
+   *
+   * @param {Object} scorepad the scorepad to redirect the user to
+   */
+  function redirectToScorepad(scorepad, sideName) {
```

Comment:
Das hier meinte ich, das steht jetzt an der falschen Stelle, weil es damit in den anderen Funktionen nicht mehr verfügbar ist. Außerdem sollte `sideName` hier als Parameter wahrscheinlich wieder raus, da wir von hier ja immer nur zu `doppelkopfSchreiben.html` weiterleiten.